rust/src/accuracy/polars/mod.rs

```diff
@@ -75,21 +77,29 @@ struct UtilitySummary {
 /// * `lazyframe` - computation from which you want to read noise scale parameters from
 /// * `alpha` - optional statistical significance to use to compute accuracy estimates
 pub fn lazyframe_utility(lazyframe: &LazyFrame, alpha: Option<f64>) -> Fallible<DataFrame> {
-    let mut utility = logical_plan_utility(&lazyframe.logical_plan, alpha)?;
+    let mut utility = logical_plan_utility(&lazyframe.logical_plan, alpha, None)?;
 
     // only include the accuracy column if alpha is passed
     if alpha.is_none() {
         utility = utility.drop("accuracy")?;
     }
+    // only include the threshold column if a threshold is set
+    if utility.column("threshold")?.is_null().all() {
+        utility = utility.drop("threshold")?;
+    }
```

Comment:
I kind of like keeping the description thin so that it is presentation-ready. As a user I'd also prefer things to error if I tried to access accuracies without setting alpha.